reference/5.1/CimCmdlets/Invoke-CimMethod.md

```diff
@@ -340,15 +336,13 @@ Accept wildcard characters: False
 
 ### -Query
 
-Specifies a query to run on the CIM server.
-A method is invoked on the instances received as a result of the query.
-
-You can specify the query dialect using the QueryDialect parameter.
+Specifies a query to run on the CIM server. A method is invoked on the instances received as a
+result of the query. You can specify the query dialect using the **QueryDialect** parameter.
 
-If the value specified contains double quotes ("), single quotes ('), or a backslash (\\), you must
-escape those characters by prefixing them with the backslash (\\) character. If the value specified
-uses the WQL LIKE operator, then you must escape the following characters by enclosing them in
-square brackets (\[\]): percent (%), underscore (_), or opening square bracket (\[).
```

Comment:
This should be a comma: (`[]`), not (`[]`):

Reply:
No. It is saying that you need to put the following characters inside square brackets: `%`, `_`, `[`.

Reply:
I made the same misread, thanks for clarifying.